### DISCUSSION TITLE
Welcome to venom-xbmc-addons Discussions!

### DISCUSSION THREAD

**Discussion by sizanic** [Kodi-vStream/venom-xbmc-addons#3443] [Annonces]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zemely** [comment]
Hello, 
it seems that venom addon lately wouldn't resume the movie or episode from where I stopped. It prompts to resume from correct point but then it starts from the beginning. Anyone else has this issue? NVidia Shield, KODI 19.3, latest Android

**Comment by sizanic** [comment]
Hello. 
These discussions are about the vStream addon. Ils the addon you use ?
Which version ?
Is a french addon. Do you speak french ?

**Comment by N2oulaz** [comment]
Salut

**Reply by sizanic** [reply]
Salut.

**Reply by TmpName** [reply]
Salut, je crois que tu as gagné le prix du deterrage de post ^^

**Comment by mIcHyAmRaNe** [comment]
Bonsoir, 
j'ai un récépteur Enigma2
il existe un plugin sur enigma2 qui a integré vstream (en gros, c'est le meme code que vstream avec quelques modifications au niveau des imports) ... sauf que ce dernier n'est pas à jour

de coup je veux le mettre à jour, mais faut que je consulte le code source de certain fichiers,  mais j'arrive pas à les trouver:  `xbmc.py` `xbmcvfs.py` `xbmcplugin.py`  `xbmcgui.py` `xbmcaddon.py` 
ils sont souvent importés dans vstream  mais je les trouve pas... et j'ai besoin de voir leurs code sources 

merci de bien vouloir m'orienter

**Reply by TmpName** [reply]
Ben ces fichiers ne sont pas dans Vstream, mais directement de kodi https://codedocs.xyz/xbmc/xbmc/group__python.html
Donc normalement si c'est basé sur kodi, meme une vielle version tu devrais les avoir aussi.

**Reply by sizanic** [reply]
ce sont les imports du code de Kodi, pour que vStream utilises Kodi pour s'integrer, ajouter des menus, afficher les informations et lancer la lecture des médias dans kodi.

**Reply by mIcHyAmRaNe** [reply]
merci pour les informations 

en effet, y avait deja les fichiers en question dans le plugin, mais datent de la renaissance ^^ ... y a des fonctions qui manquent

ma 1ere tentative est vouée à l’échec, vstream a disparu de plugin à cause la fonction translatePath 

bref , je vais devoir me creuser la tete pour trouver les problemes et les regler
chose qui sera pas facile, vu que c est ma 1ere fois sur enigma2 et j'ai pas trouvé quelqu'un pour m'expliquer comment ils testent les plugins et les debug

**Reply by sizanic** [reply]
Amazon FireStick 4K Max => 40€uros ! 😉

**Comment by aioros57** [comment]
Bonjour sizanic ,je peux parler en Français   pas de souci .Je trouve que  Vstream   a beaucoup de souci  depuis  Kodi nexus 20.1 ,tous es à jours  Vstream   et autres  quand  je vais sur des sources  film ou sport  il y a message  cloudfare   j'ai suivi  ce tutoriel  https://github.com/Kodi-vStream/venom-xbmc-addons/wiki/Comment-passer-CloudFlare[](url)  rien ne change  .
Faut-il revenir sur un kodi antérieur  comme  matrix  19.x, ou désinstaller  kodi  nexus  actuel, la réinstaller  propre comme j'ai  modifié des scripts par rapport tutoriel du lien comme expliqué.
merci d'avance

**Reply by sizanic** [reply]
Bonjour,
cloudflare est une protection mise en place par le site internet qui propose les liens des vidéos.

Quelque soit la version de kodi et la version de vStream, cette protection sera toujours là, ainsi que le message d'erreur.

L'impression que cela fonctionnait mieux avec la version 19 vient qu'a l'époque de la version 19, le site n'avait pas cette protection cloudflare. Ce n'était pas grâce à la version 19.


Si tu n'arrives pas à passer la protection Cloudflare, désactive les sources qui utilisent cette protection (en suivant les messages d'erreurs), et utilise seulement les autres.

Penses également à changer les parametres DNS de ton réseau (ou installe l'extension DNSPython depuis le chemin de depot de vStream). Cela redonnera l'acces à certains sources bloquées par ton opérateur internet.

D'après cet article, plus de 40 sources devraient permettre l'accès à des films :
https://lesalkodiques.com/groupes/vstream/forum/sujet/plus-beaucoup-de-choix-de-serveur/

La source "Wiflix" est reconnue comme étant la source gratuite la plus complète.
La source "Pastebin" est la source la plus complète qui nécessite un compte premium.

**Comment by TmpName** [comment]
Pour moi cela ne changera rien.
Il vaut mieux corriger ton probleme, si tu en as vraiment un (logs ?)
